webpack/generate-chunks-map-plugin.js

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
Can we move this file and `bin/pot-dist-references.js` to the `scripts` (or `scripts/pot`) directory, so that there is only one top-level directory in the project for build-related files? Would clear it up a bit.

Reply:
Good idea! Updated here: https://github.com/Automattic/sensei/pull/3493/commits/3f8f87ba5ebf7fcd3a784db59e975be945b67407

Reply:
I kept the pot script without a separate folder as it's only a single file. And I created a folder for Webpack plugins and related, where probably we can have more files in the future.